### ISSUE TITLE
Feature Request:  Ability to add threads to round objects such as body tubes, fin cans, etc.

### ISSUE THREAD

**Issue by livetoski78** [davesrocketshop/Rocket#66]
This would greatly ease trying to make threaded rocket parts, as typical thread creation in FreeCAD is quite an involved process (and one I haven't figured out yet)

**Comment by davesrocketshop** [comment]
Interesting application. The hard part will be generalizing the UI

**Comment by livetoski78** [comment]
Indeed.  From what I’ve compiled so far from the documentation, it involves sketching a triangle shape, helixes, and sweeping objects, then some cutting and pasting the ends on.  I know there are guys out there doing it, but it’s way beyond my rudimentary CAD skills.  Ideally, I want to create a screw in electronics bay in the nose cone for my featherweight trackers.  Right now, it’s a hack job using a printed plate and screws.  Also, screw together body tubes would be nice to break down larger rockets for transport and easier printing (I can do up to 12” length, currently).-Eric====================Sent from my iPhoneOn Nov 24, 2023, at 23:21, David Carter ***@***.***> wrote:﻿
Interesting application. The hard part will be generalizing the UI
